### PR TITLE
Added RHEL package list for installation instructions

### PR DESCRIPTION
**Pull request by czhanacek**
Signed-off-by: czhanacek <charlie@czhanacek.com>
@brian-brazil

### CODE REVIEW

File: generator/README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ Due to the dynamic dependency on NetSNMP, you must build the generator yourself.
 
 ```
 sudo apt-get install build-essential libsnmp-dev # Debian-based distros
+OR
```
Comment (brian-brazil) [inline generator/README.md]: This reads like it's a command, use a comment like above.

**Comment by czhanacek** [comment]
Both issues should be fixed in my latest commit

File: generator/README.md (hunk 1 above)
Comment (brian-brazil) [reply generator/README.md]: There's still the OR here, and no comment saying the other is for RH distros

**Comment by czhanacek** [comment]
Forgot to add RHEL comment, it's in there now.

File: generator/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Due to the dynamic dependency on NetSNMP, you must build the generator yourself.
 ```
 sudo apt-get install build-essential libsnmp-dev # Debian-based distros
 # OR
```
Comment (brian-brazil) [inline generator/README.md]: Can you remove this line?

Comment (brian-brazil) [reply generator/README.md]: The line with the OR

Comment (czhanacek) [reply generator/README.md]: What if we did
# Debian-based distros
apt-get command
# RHEL-based distros
yum command

That way, it doesn't read like both commands should be executed

Comment (SuperQ) [reply generator/README.md]: That sounds reasonable to me.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**